common/app/views/main.scala.html

```diff
@@ -5,16 +5,18 @@
 @import views.support.{Commercial, RenderClasses}
 @import play.api.Mode.Dev
 
-@headerAndTopAds(showAdverts: Boolean, edition: Edition) = {
-    @if(!page.metadata.shouldHideHeaderAndTopAds) {
-        <div id="bannerandheader">
-            @if(showAdverts) {
-                @fragments.commercial.topBanner(page.metadata)
-            }
-            @fragments.header(page)
-        </div>
-        <div id="maincontent" tabindex="0"></div>
+@headerAndTopAds(showAdverts: Boolean, edition: Edition, content: Option[model.ContentType]) = {
+  @if(!page.metadata.shouldHideHeaderAndTopAds) {
+    @defining(showAdverts && (content.isEmpty || content.exists(content => !content.tags.isTheMinuteArticle))) { showTopSlot =>
```

Comment:
hmm, can `!content.exists(_.tags.isTheMinuteArticle)` be equivalent to the whole expression after the `&&`?